spikingjelly/activation_based/triton_kernel/flex_sn_inductor/__init__.py

```
@@ -1,9 +1,27 @@
+        FlexSNScan,
+        eager_scan,
+        eager_scan_final_state,
+        flex_sn_scan,
+        lowerable_scan,
+        lowerable_scan_available,
+        lowerable_scan_final_state,
+        lowerable_while_loop_available,
+        lowerable_while_loop_scan,
+        lowerable_while_loop_scan_final_state,
+    )
+except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)